mandatory/2-function-creation.js

```diff
@@ -15,26 +21,62 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200) {
+    totalPrice = totalPrice - ((totalPrice * 10) / 100);
+    return totalPrice;
+  } else {
+    totalPrice = totalPrice - ((totalPrice * 5) / 100);
+    return totalPrice;
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let number = 1;
+  let step = 1;
+
+  while (number < limit) {
+    console.log(number);
+    number += 2;
+    step += 1;
```

Comment:
what is this step for? 🤔

Reply:
The first I used another way, but I Changed it and forgot to remove this variable😅 I'll correct it.